index.js

```diff
@@ -504,6 +580,13 @@ const ContainerService = {
         userContextId = args.userContextId;
```

Comment:
Not sure we have to reject? I'll change to set a default value if `sources` isn't in the `args` in case I'm missing a source of calling code.